docs/installation_guide_ova.md

```diff
@@ -175,3 +175,18 @@ If you want to change the properties of Harbor, follow the below steps:
  * **MySQL Root Password**: Change it by logging in the virtual appliance and doing it in the Linux operating system.  
  * **Networking Properties**: Visit `https://harbor_ip_address:5480`, log in with root/password of your virtual appliance and modify networking properties. Reboot the system after modification to ensure Harbor's self-signed certificate gets updated.  
  
+## Troubleshooting
+
+### Log collection
+When meet some issues that you can not fix by yourself, you may need the logs of Harbor to send to someone else to check what happened. Use the following steps to do it:  
```

Comment:
Replace with: 
For diagnosis purpose, logs can be collected by the following steps: